### ISSUE TITLE
Password field seems not selectable nor editable

### ISSUE THREAD

**Issue by mquinson** [YunoHost-Apps/hotspot_ynh#25]
During the installation process, a wifi password is generated, but the field containing it seems neither editable nor selectable. This is because the default value is proposed when there is no value in the field, so you cannot select that value. Instead, you have to clic the field, and start typing without removing the previous value. That's a bit disturbing I must say.

Thanks for this huge project, I love you all.

**Comment by jvaubourg** [comment]
Interesting usability problem :).

The Wifi Hotspot does not generate a default password. All the values displayed in gray inside the fields are not default values, but example values (*placeholders*). So, *VhegT8oev0jZI* is just an example, and it's the same value for all the installations. If you Save & Reload the page without filling this field, you will have an error because the field is empty, without any default value.

But, with your issue, I understand this could be confused for the users. Do you think this field should be totally empty by default? And for the other fields (e.g. Server Name also has a *placeholder* with *vpn.ldn-fai.net* when empty, and it's not a default value)?

**Comment by mquinson** [comment]
I definitely think that an empty field is preferable to the current situation. I gave up any hope of configuring my password, but kept the app running to open that bug. So I discovered how it worked in that process, after abandoning.

In the same spirit, the application (or another of the briquenet bundle?) have a yes/no configuration item. That's a string for now while I saw another application proposing a yes/no combobox where you cannot type yse or anything wrong... Less critical, but still anoying
